dockstore-webservice/src/main/java/io/dockstore/webservice/resources/EntryResource.java

```diff
@@ -421,6 +433,101 @@ public Entry setDiscourseTopic(@ApiParam(hidden = true) @Parameter(hidden = true
         return createAndSetDiscourseTopic(id);
     }
 
+    @Path("/updateLanguageVersions")
+    @RolesAllowed("admin")
+    @Deprecated
```

Comment:
codacy has a good suggestion here to document when we expect to delete this in the annotation itself (in case we forget, get distracted, etc.)

Reply:
I'm actually wondering if we should keep this around, just in case. Now I'm using `@Deprecated` to sort of hide it, but now I realize there actually is a `@Hidden` annotation, so I should use that, if we keep it. See my other comment for the rationale on whether to keep it.

Reply:
Ended up removing the annotation (see other comment).